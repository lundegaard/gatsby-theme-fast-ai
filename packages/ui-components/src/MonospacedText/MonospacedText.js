import React from 'react';

import Text from '../Text';

const MonospacedText = props => <Text fontFamily="mono" letterSpacing="mono" {...props} />;

export default MonospacedText;
