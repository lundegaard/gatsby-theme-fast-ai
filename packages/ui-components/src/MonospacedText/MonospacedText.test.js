import React from 'react';
import { shallow } from 'enzyme';

import MonospacedText from './MonospacedText';

describe('<MonospacedText />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<MonospacedText {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
