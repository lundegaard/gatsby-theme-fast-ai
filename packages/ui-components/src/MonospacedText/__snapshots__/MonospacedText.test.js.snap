// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MonospacedText /> matches snapshot when shallowly rendered 1`] = `
<Text
  fontFamily="mono"
  letterSpacing="mono"
/>
`;
